src/os/filestore/HashIndex.cc

```diff
@@ -396,7 +396,7 @@ int HashIndex::read_settings() {
   if (r == -ENODATA)
     return 0;
   if (r < 0) {
-    derr << __func__ << " error reading settings: " << cpp_strerror(r) << dendl;
+    dout(0) << __func__ << " error reading settings: " << cpp_strerror(r) << dendl;
```

Comment:
@tchaikov 
Thanx for being persistent... Is was an error in the headers.
Perhaps still something leftover from cleaning up.

It is again a `ENODATA` problem ....
No `compat.h` included, so `ENODATA` has the wrong (boost) value.